asset-management-labs/improving-data-quality.ipynb

```diff
@@ -83,9 +83,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
output needs to be cleared before checking file in

Reply:
Hi Nigel - I added a line in the config file for an xtdb option using what was in common - but looks like I didn't upload the right file. I'll try again.

Reply:
The notebook still includes output, it needs to be saved without (Edit->Clear all outputs)